pipe/pipe.go

```diff
@@ -0,0 +1,123 @@
+// Adapted from https://github.com/rheinardkorf/go-concurrency/blob/master/11_pipeline_complex/pipe/pipe.go
+package pipe
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"github.com/wptide/pkg/process"
```

Comment:
Completely possible to use relative paths... but there are trade-offs yes.

E.g. If we were to move pipe to another package altogether, then it would be relative anymore. Or if we decide to restructure our tree.  It is not common go practice to use relative paths.... paths should by convention always relate to the structure of your GOPATH, except for internal Go packages which relate to GOROOT.